<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelaunayClass</class>
 <widget class="QDialog" name="DelaunayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delaunay三角网</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>输出到</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>容差</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择图层</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>是否只生成边</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
